src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,259 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.File;
+
+public class Duke {
+
+    public static final int MAX_TASK = 100;
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_DELETE = "delete";
+    public static final String COMMAND_MARK = "mark";
+    public static final String COMMAND_UNMARK = "unmark";
+    public static final String COMMAND_BYE = "bye";
+    public static final String COMMAND_EVENT = "event";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_TODO = "todo";
+    public static final File filePath = new File("src/main/java/duke/task/Hage.txt");
+
+
+    private static int itemNum;
+    private static String[] dateArray;
+    private static String[] eventArray;
```

Comment:
Consider using [plural form](https://se-education.org/guides/conventions/java/basic.html#naming) for naming a collection of objects - e.g. `dates`, `events`.